packages/backend-core/src/users/lookup.ts

```diff
@@ -26,6 +27,9 @@ export async function searchExistingEmails(emails: string[]) {
   const existingAccounts = await getExistingAccounts(emails)
   matchedEmails.push(...existingAccounts.map(account => account.email))
 
+  const invitedEmails = await getExistingInvites(emails)
+  matchedEmails.push(...invitedEmails.map(invite => invite.email))
```

Comment:
This is the key change that fixes the bug. It was wanting to do this here that made me move the invite / password reset functionality into `backend-core`.